server/auth_test.go

```diff
@@ -1496,11 +1543,20 @@ func TestAuthorizedUser(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			var authorized bool
+			var hasServerCtx bool
+			var hasSuperAdminCtx bool
+			var hasOrgCtx bool
```

Comment:
we spell out Organization everywhere else. would prefer that here for consistency: `hasOrganizationContext`